src/Orleans.Transactions/DistributedTM/TransactionAgentStatisticsReporter.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Options;
+using Orleans.Configuration;
+using Orleans.Runtime;
+using Orleans.Transactions.Abstractions;
+
+namespace Orleans.Transactions
+{
+    public class TransactionAgentStatisticsReporter : ILifecycleParticipant<ISiloLifecycle>
+    {
+        private const string TransactionsStartedTotalMetric = "TransactionAgent.TransactionsStarted_Total";
+        private const string TransactionsStartedPerSecondMetric = "TransactionAgent.TransactionsStarted_PerSecond";
+
+        private const string SuccessfulTransactionsTotalMetric = "TransactionAgent.SuccessfulTransactions_Total";
+        private const string SuccessfulTransactionsPerSecondMetric = "TransactionAgent.SuccessfulTransactions_PerSecond";
+
+        private const string FailedTransactionsTotalMetric = "TransactionAgent.FailedTransactions_Total";
+        private const string FailedTransactionsPerSecondMetric = "TransactionAgent.FailedTransactions_PerSecond";
+
+        private const string ThrottledTransactionsTotalMetric = "TransactionAgent.ThrottledTransactions_Total";
+        private const string ThrottledTransactionsPerSecondMetric = "TransactionAgent.ThrottledTransactions_PerSecond";
```

Comment:
Convention is to use only `.` as separators - see [StatisticNames.cs](https://github.com/dotnet/orleans/blob/3d2bf21e1f43fb24ec8bcb81aae9e6165c797a3a/src/Orleans.Core/Statistics/StatisticNames.cs#L107)